python/py-gpxpy/Portfile

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim: fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-gpxpy
+version             1.6.2
+revision            0
+categories-append   devel
+license             Apache-2
+supported_archs     noarch
+platforms           {darwin any}
+
+python.versions     38 39 310 311 312
```

Comment:
Python 3.8 is almost EOL, so we certainly don't want to add that here anymore. Typically for new ports I'd suggest to just support the latest or last two versions, unless there are good reasons to go back to earlier versions.